specs/lib/gltfPipelineSpec.js

```diff
@@ -118,9 +118,13 @@ describe('gltfPipeline', function() {
     it('will write sources from JSON', function(done) {
         var options = {};
         readGltf(gltfEmbeddedPath, options, function (gltf) {
-            var initialUri = gltf.buffers.CesiumTexturedBoxTest.uri;
+            var firstBufferId = Object.keys(gltf.buffers)[0];
+            var testBuffer = gltf.buffers[firstBufferId];
+            var initialUri = testBuffer.uri;
```

Comment:
I don't think you need to do this here, just after processJSON gets run, right? You do know the initial bufferId is CesiumTexturedBoxTest.